ixwebsocket/IXWebSocketTransport.cpp

```diff
@@ -444,12 +488,16 @@ namespace ix
             else if (ws.opcode == wsheader_type::PING)
             {
                 unmaskReceiveBuffer(ws);
+
                 std::string pingData(_rxbuf.begin()+ws.header_size,
-                                     _rxbuf.begin()+ws.header_size + (size_t) ws.N);
+                                         _rxbuf.begin()+ws.header_size + (size_t) ws.N);
```

Comment:
Was that line not indented properly ? New indent doesn't look correct on github.

Reply:
Maybe, I’ll have to look